.github/workflows/ubuntu.yml

```diff
@@ -12,7 +12,9 @@ jobs:
     steps:
     # WxWidgets
     - name: install dependencies for wxWidgets
-      run: sudo apt-get install build-essential cmake git libedit-dev libgtk-3-dev libhunspell-dev libsqlite3-dev libssh-dev pkg-config xterm
+      run: |
+        sudo apt-get update
+        sudo apt-get install build-essential cmake git libedit-dev libgtk-3-dev libhunspell-dev libsqlite3-dev libssh-dev pkg-config xterm
```

Comment:
Better user `-y` to avoid extra questions?
`sudo apt-get install -y ...`

Reply:
I dont know, not familiar.